tests/tests_suite_test.go

```diff
@@ -471,6 +470,13 @@ func getSsp() *sspv1beta2.SSP {
 	return foundSsp
 }
 
+func getSspV1Beta3() *sspv1beta3.SSP {
```

Comment:
Can we already switch to v1beta3 by default in tests?

Reply:
Let's not do it yet. We can do it in 4.19.

Reply:
Ok